src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -160,6 +160,7 @@ public void onResume() {
     super.onResume();
 
     updateReminders();
+    DozeReminder.maybeAskDirectly(getActivity());
```

Comment:
i did not tested that, however, onResume is called whenever the activity is reused - i think it is not needed to check for the the DozeReminder in these cases. also, not sure why this goes to the fragment.

maybe ConversationListActivity.onStart() is a better place (ConversationListActivity.onCreate() where we add device-messages, however, is too soon as the activity is not visible)